aws/examples/simple/main.tf

```diff
@@ -325,6 +419,60 @@ locals {
     local.materialize_instance_namespace,
     local.materialize_instance_name
   )
+
+  ami_selector_terms = [{ "alias" : "bottlerocket@latest" }]
```

Comment:
Are we good using `@latest` here or should we pin to a specific version?

Reply:
As this is just the example, I think latest is fine. We'd have to update it frequently to ensure we don't force an insecure version otherwise.